CODEOWNERS

```diff
@@ -0,0 +1,101 @@
+# ==============================================================================
+#                          Bitcoin Core CODEOWNERS
+# ==============================================================================
+
+# Configuration of code ownership and review approvals for the bitcoin/bitcoin
+# repo.
+
+# Order is important; the last matching pattern takes the most precedence.
+# More info on how this file works can be found at:
+# https://help.github.com/articles/about-codeowners/
+
+# This file is called CODEOWNERS because it is a magic file for GitHub to
+# automatically suggest reviewers. In this project's case, the names below
+# should be thought of as code reviewers rather than owners. Regular
+# contributors are free to add their names to specific directories or files
+# provided that they are willing to provide a review when automatically
+# assigned.
+
+# Information on the decision making process for pull requests can be found
+# in the "Decision Making Process" section of the CONTRIBUTING.md doc.
+
+
+# Maintainers
+#   @laanwj
+#   @sipa
+#   @fanquake
+#   @jonasschnelli
+#   @marcofalke
+#   @meshcollider
+
+
+# Build system
+/configure.ac                       @dongcarl
+/build-aux/                         @dongcarl
+/depends/                           @dongcarl
+*.am                                @dongcarl
+Makefile.*                          @dongcarl
+
+# Test framework
+/src/test/                          @marcofalke
+/test/                              @marcofalke
+/ci/                                @marcofalke
+
+# Fuzzing
+/src/test/fuzz/                     @practicalswift
+/doc/fuzzing.md                     @practicalswift
+
+# GUI
+/src/qt/                            @jonasschnelli
+
+# Wallet
+/src/wallet/                        @meshcollider
+
+# Descriptors
+*descriptor*                        @achow101 @sipa
+
+# MSVC build system
+/build_msvc/                        @sipsorcery
+
+# scripts/linter
+*.sh                                @practicalswift
+/test/lint/                         @practicalswift
+/ci/lint/                           @practicalswift
+
+# Bech32
+/src/bech32.*                       @sipa
+/src/bench/bech32.*                 @sipa
+
+# PSBT
+/src/psbt*                          @achow101
+/src/node/psbt*                     @achow101
+/doc/psbt.md                        @achow101
+
+# Consensus
+/src/consensus/                     @sipa
+/src/consensus/tx_verify.*          @jtimon
+/src/consensus/validation.h         @TheBlueMatt
+/src/consensus/tx_check.*           @jnewbery
+
+# Docs
```

Comment:
I think this section should be in the front. If a PR touches code and one of these files I think the code reviewers should be the ones getting tagged. Meaning I think most of the time that is the better fit rather than the other way around.

Reply:
Makes sense. Moved.